.gitignore

```diff
@@ -101,3 +101,9 @@ test-output/
 ### AppEngine template
 # Google App Engine generated folder
 appengine-generated/
+
+# Compilation tests
+url-gen/src/test/kotlin/com/cloudinary/CompilationTest.kt
+devTools/error.json
+devTools/success.json
+devTools/results.json
```

Comment:
Please set up your IDE to put new line at the end of file on save. These red signs really distract. thanks!

Reply:
I'm strongly against new lines in end of files 😄 😄